includes/Core/Assets/Assets.php

```diff
@@ -666,6 +668,7 @@ private function get_inline_base_data() {
 			'siteName'         => get_bloginfo( 'name' ),
 			'enabledFeatures'  => Feature_Flags::get_enabled_features(),
 			'webStoriesActive' => defined( 'WEBSTORIES_VERSION' ),
+			'multipleUsers'    => $users['total_users'] > 1,
```

Comment:
This is not specific enough to meet the ACs which states that "the notice is only shown if more than one WordPress user can activate/use Site Kit" where as this over-simplifying that to whether or not the site has more than one user.

For now this is limited to administrators but ideally we would scope this based on capabilities rather than a specific role as this will change in the future. Unfortunately, WP doesn't provide a way to query by capability, and Site Kit's capabilities are "virtual" anyways so we would have to query for the underlying primitive capability (`manage_options` for `Permissions::AUTHENTICATE`) but this may miss users who actually have the Site Kit capability, but not the primitive. All that to say, I think we're stuck querying by role right now 😄 

We probably should not however loop through queried users and check `user_can( $user_id, Permissions::AUTHENTICATE )` since this could be potentially expensive with a large number of users who don't have the permission.

Reply:
See above; let's use a new `Has_Multiple_Admins` class; we can then expose it with the site connection in its REST endpoint, alongside the existing `hasConnectedAdmins` value.